examples/tts/conf/spectrogram-enhancer.yaml

```diff
@@ -0,0 +1,86 @@
+name: "spectrogram-enhancer"
+
+model:
+  n_bands: 80
+  latent_dim: 192
+  style_depth: 4
+  network_capacity: 16
+  mixed_prob: 0.9
+  fmap_max: 192
+
+  generator:
+    _target_: "nemo.collections.tts.modules.spectrogram_enhancer.Generator"  
+    n_bands: ${model.n_bands}
+    latent_dim: ${model.latent_dim}
+    network_capacity: ${model.network_capacity}
+    style_depth: ${model.style_depth}
+    fmap_max: ${model.fmap_max}
+
+  discriminator:
+    _target_: "nemo.collections.tts.modules.spectrogram_enhancer.Discriminator"
+    n_bands: ${model.n_bands}
+    network_capacity: ${model.network_capacity}
+    fmap_max: ${model.fmap_max}
+
+  consistency_loss_weight: 10.0  # somewhere in [1., 100.], less for clean datasets, higher for noisier
+  gradient_penalty_loss_weight: 10.0  # read stylegan papers before changing
+  gradient_penalty_loss_every_n_steps: 4
+
+  # Spectrogram values range, calculated over your dataset with matching STFT parameters.
+  # Needed for treating spectrograms as images with pixel values around [0, 1].
+  # For LibriTTS, you can try [-13.18, 4.78]
+  spectrogram_min_value: ???
+  spectrogram_max_value: ???
+
+  train_ds:
+    dataset:
+      _target_: "nemo.collections.tts.torch.data.PairedRealFakeSpectrogramsDataset"
+      manifest_filepath: "/home/user/train-clean-100.manifest_mel.json"
```

Comment:
Let's not put paths over here. Change this to a mandatory field i.e. ???

Reply:
Good catch, thank you

This is a spillover from my test docker environment, definitely supposed to be `???`

Reply:
Fixed